pkg/controllers/applicationfailover/common.go

```diff
@@ -210,11 +210,12 @@ func buildTaskOptions(failoverBehavior *policyv1alpha1.ApplicationFailoverBehavi
 	}
 
 	switch failoverBehavior.PurgeMode {
-	case policyv1alpha1.Graciously:
+	//nolint:staticcheck
```

Comment:
add a comment to explain why to suppress this check.

Reply:
updated